bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/obsrecord/PrivateVisitList.java

```diff
@@ -22,7 +23,7 @@
  */
 final class PrivateVisitList implements Serializable {
     private static final long serialVersionUID = -2686488059242714341L;
-    
+
```

Comment:
I verified that serialization won't be an issue which stands to reason since the same APIs were previously updated to pass in the instrument.